deepspeed/inference/engine.py

```diff
@@ -151,6 +151,13 @@ def __init__(self, model, config):
             assert pkg_version.parse(torch.__version__) >= pkg_version.parse("1.10"), \
                 "If you want to use cuda graph, please upgrade torch to at least v1.10"
 
+        # Check if model passed to engine is loaded w/ meta tensors, in which case
+        # kernel injection must be enabled.
```

Comment:
Please update the comment to add why this only works for HF models.

Reply:
Added a note saying that the device type is sourced assuming a Hugging Face hierarchy.